physics/module_bl_mynn.F90

```diff
@@ -736,51 +737,14 @@ SUBROUTINE mynn_bl_driver(            &
                 sqc(k)=sqc3D(i,k) !/(1.+qv(i,k))
                 sqv(k)=sqv3D(i,k) !/(1.+qv(i,k))
                 thetav(k)=th(i,k)*(1.+0.608*sqv(k))
```

Comment:
opportunity to use p608 rather than redefined constant

Reply:
indeed.